tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -16,7 +16,7 @@ Names:
 [103..106): Int => _root_.scala.Int#
```

Comment:
Can we add a new expect test that covers all of cases of `Type`? A single file containing annotated type, existentials, etc.

Reply:
Will do.